posthog/test/test_client.py

```diff
@@ -417,7 +417,7 @@ def test_feature_enabled_simple(self, patch_get, patch_decide):
     @mock.patch("posthog.client.decide")
     @mock.patch("posthog.client.get")
     def test_feature_enabled_simple_is_false(self, patch_get, patch_decide):
-        client = Client(TEST_API_KEY)
+        client = Client(TEST_API_KEY, personal_api_key="test")
```

Comment:
why'd you have to add this here? 🤔

Reply:
Ah ha, I was blindly adding the `personal_api_key` to all tests which mocked `client.decide`. No need in here.